pdf.go

```diff
@@ -34,6 +35,15 @@ func writeLogo(pdf *gopdf.GoPdf, logo string, from string) {
 	_ = pdf.SetFont("Inter", "", 12)
 	pdf.SetTextColor(55, 55, 55)
 	_ = pdf.Cell(nil, from)
+	pdf.Br(18)
```

Comment:
This should probably be extracted into another function and called here?

Reply:
updated in https://github.com/maaslalani/invoice/pull/21/commits/67ff7f63228b4f93a01a472ed7c57304a8240c16 commit